### DISCUSSION TITLE
There are outstanding issues with your GitHub account: Please contact GitHub Support for more information.

### DISCUSSION THREAD

**Discussion by kroblus** [community/community#67894] [GitHub Education]
### Hi everyone,

This sentence stopped me after I applied for the GitHub student package: There are outstanding issues with your GitHub account: Please contact GitHub Support for more information.

**Comment by Zerotask** [comment]
You can contact the GitHub support here: https://support.github.com/request?tags=docs-generic

**Comment by amy011872** [comment]
Hi, I have encountered the same issue, and it is the contacting GitHub support that leads me here!
What else should I do in GitHub support other than posting an issue? 
Thanks in advance!
